llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -279,8 +280,10 @@ class Triple {
     Amplification,
     OpenCL,
     OpenHOS,
+    Kernel,
```

Comment:
Why is a generic term `Kernel` added? I am concerned that it would cause confusion to users of other OSes.

Does your OS need a different target triple for kernel development? Note that almost all other OSes don't make the target triple distinction.